src/client.rs

```diff
@@ -214,6 +249,36 @@ impl<T: Authenticator + Clone> Session<T> {
         try!(tcp.write(query_frame.as_slice()));
         return parse_frame(tcp, &self.cdrs.compressor);
     }
+
+    /// The method makes a request to DB Server to execute a query provided in `query` argument.
+    /// you can build the query with QueryBuilder
+    /// let qb = QueryBuilder::new().query("select * from emp").consistency(Consistency::One).page_size(Some(4));
+    /// session.query_with_builder(qb);
+    ///
+    pub fn query_with_builder(&self,qb : QueryBuilder) -> error::Result<Frame> {
```

Comment:
Hmm how do you typically do formatting? I do the
'Cargo fmt' before committing may be cargo clippy might automatically do this ?

I'll give it a shot